### PR TITLE
small storage changes

### PR DESCRIPTION
**Pull request by etbe**

### CODE REVIEW

File: policy/modules/kernel/files.if [hunk 1]
```diff
@@ -1426,6 +1426,25 @@ interface(`files_unmount_all_file_type_fs',`
...
+## <summary>
+##	watch all directories of file_type
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`files_watch_all_file_type_dir',`
```
Comment (pebenito) [inline policy/modules/kernel/files.if:1439]: files_watch_all_dirs

File: policy/modules/services/smartmon.if [hunk 2]
```diff
@@ -56,3 +56,24 @@ interface(`smartmon_admin',`
...
+##	</summary>
+## </param>
+## <rolecap/>
+#
+interface(`fsdaemon_read_lib',`
+	gen_require(`
+		type fsdaemon_var_lib_t;
+	')
+
+	allow $1 fsdaemon_var_lib_t:dir search;
```
Comment (pebenito) [inline policy/modules/services/smartmon.if:76]: please use `search_dir_perms`, alternatively switch the implementation to use `read_files_pattern`.

**Review by pebenito — CHANGES_REQUESTED** [review]

File: policy/modules/kernel/files.if (hunk 1 above)
Comment (etbe) [reply policy/modules/kernel/files.if]: done

File: policy/modules/services/smartmon.if (hunk 2 above)
Comment (etbe) [reply policy/modules/services/smartmon.if]: done

**Review by pebenito — APPROVED** [review]

**Pull request merged**